uefi/gberegion.go

```diff
@@ -18,6 +22,19 @@ func NewGBERegion(buf []byte, r *Region) (*GBERegion, error) {
 	return &gbe, nil
 }
 
+// Validate Region
+func (gbe *GBERegion) Validate() []error {
+	// TODO: Add more verification if needed.
+	errs := make([]error, 0)
+	if gbe.Position == nil {
+		errs = append(errs, fmt.Errorf("GBERegion position is nil"))
```

Comment:
nit: I have no strong opinions and don't know if there's any significant difference, but since there's no format this could be `errors.New` instead of `fmt.Errorf` (feel free to ignore)

Reply:
done!